docs/_data/members.yml

```diff
@@ -0,0 +1,75 @@
+- name: Hong Ge
+  photo: https://yebai.github.io/images/hg344-pr.jpg
+  github: yebai
+  institution: University of Cambridge
+  website: http://mlg.eng.cam.ac.uk/hong/
+  email: hg344@cam.ac.uk
+
+- name: Kai Xu
+  photo: placeholder.png
+  github: xukai92
+  institution: University of Edinburgh
+  website: https://xuk.ai/
+  email: your-email@something.com
+  linkedin: linked-page
+  twitter: twitter-handle
+
+- name: Martin Trapp
+  photo: placeholder.png
```

Comment:
https://martintdotblog.files.wordpress.com/2018/06/photo.jpg

Do you scale all images?

Reply:
Yeah, they're scaled.